ts/navigation/TabNavigator.tsx

```diff
@@ -80,6 +91,14 @@ export const MainTabNavigator = () => {
           tabBarIcon: ({ color }) => <WalletTabIcon color={color} />
         }}
       />
+      {/* <Tab.Screen
+        name={ROUTES.PAYMENT_SCAN_QR_CODE}
+        component={ScanQrCodeScreen}
+        options={{
+          title: I18n.t("global.navigator.scan"),
+          tabBarIcon: ({ color }) => <ScanTabIcon color={color} />
+        }}
+      /> */}
```

Comment:
It should be better to remove that block comment or at least leave an explaining comment along with it. What do you think?

Reply:
Instead of leaving commented code into the application, why do not we use a feature flag to disable it?